src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,29 @@
+package lotto.domain;
+
+import java.util.EnumMap;
+import java.util.List;
+
+public class Lottos {
+
+    private final List<Lotto> lottos;
+
+    public Lottos(List<Lotto> lottos) {
+        this.lottos = lottos;
+    }
+
+    public EnumMap<Rank, Integer> matchResult(WinningNumbers winningNumbers) {
+        EnumMap<Rank, Integer> result = new EnumMap<>(Rank.class);
+        for (Rank rank : Rank.values()) {
+            result.put(rank, 0);
+        }
+        for (Lotto lotto : lottos) {
+            Rank rank = winningNumbers.match(lotto);
+            result.put(rank, result.get(rank) + 1);
+        }
+        return result;
+    }
+
+    public List<Lotto> getLottos() {
+        return lottos;
```

Comment:
코드리뷰 하다보면 다들 불변성을 위해 방어적 복사를 많이 적용 하시더라구요 !
```suggestion
        return List.copyOf(lottos);
```

Reply:
다른 부분에서는 방어적 복사를 적용했는데 이 부분을 놓친 것 같습니다.
앞으로는 일관성 있게 적용될 수 있도록 꼼꼼히 확인하겠습니다. 피드백 감사드립니다!